### ISSUE TITLE
Title module doesn't trim whitespace

### ISSUE THREAD

**Issue by zuzak** [zuzakistan/civilservant#292]
Elements in HTML can have weird whitespace. Most user agents trim the whitespace before rendering to the user, but civilservant does not:
```
10:22:58 < human> https://uk.reuters.com/article/britain-politics-fire/british-parliament-evacuated-due-to-fire-alert-reuters-witness-idUKL9N22X01E
10:22:58 <+bot>                 British parliament evacuated due to fire alert - Reuters witness - Reuters
```

**Comment by zuzak** [comment]
Fixed in #304

**Comment by zuzak** [comment]
Well, that trims whitespace on the outside, but I bet there's some website somewhere that puts the whitespace in the middle just to confuse you
